document-generator-interfaces/src/main/java/uk/gov/companieshouse/document/generator/interfaces/GetDocumentInfo.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.companieshouse.document.generator.interfaces;
+
+import uk.gov.companieshouse.document.generator.interfaces.model.DocumentInfo;
+
+public interface GetDocumentInfo {
+    DocumentInfo getDocumentInfo();
```

Comment:
Can you add a JavaDoc to this interface please to help developers understand the contract/intention of this interface please.